src/layer/x86/convolution_x86.cpp

```diff
@@ -120,55 +119,56 @@ int Convolution_x86::create_pipeline(const Option& opt)
 
     if (opt.use_int8_inference && weight_data.elemsize == (size_t)1u)
     {
+        support_packing = false;
         return create_pipeline_int8_x86(opt);
     }
 
     int kernel_size = kernel_w * kernel_h;
     int num_input = weight_data_size / kernel_size / num_output;
 
     use_winograd3x3 = false;
-
-    if ((!support_packing || !opt.use_packing_layout) && kernel_w == kernel_h && dilation_w != 1 && dilation_h == dilation_w && stride_w == 1 && stride_h == 1)
-    {
-        convolution_dilation1 = ncnn::create_layer(ncnn::LayerType::Convolution);
-
-        // set param
-        ncnn::ParamDict pd;
-        pd.set(0, num_output); // num_output
-        pd.set(1, kernel_w);
-        pd.set(11, kernel_h);
-        pd.set(2, 1);
-        pd.set(12, 1);
-        pd.set(3, 1);  // stride_w
-        pd.set(13, 1); // stride_h
-        pd.set(4, 0);  // pad_w
-        pd.set(14, 0); // pad_h
-        pd.set(5, bias_term);
-        pd.set(6, weight_data_size);
-
-        convolution_dilation1->load_param(pd);
-
-        // set weights
-        if (bias_term)
-        {
-            ncnn::Mat weights[2];
-            weights[0] = weight_data;
-            weights[1] = bias_data;
-
-            convolution_dilation1->load_model(ModelBinFromMatArray(weights));
-        }
-        else
-        {
-            ncnn::Mat weights[1];
-            weights[0] = weight_data;
-
-            convolution_dilation1->load_model(ModelBinFromMatArray(weights));
-        }
-
-        convolution_dilation1->create_pipeline(opt);
-
-        return 0;
-    }
+    // TODO: FIX ME
+    // if ((!support_packing || !opt.use_packing_layout) && kernel_w == kernel_h && dilation_w != 1 && dilation_h == dilation_w && stride_w == 1 && stride_h == 1)
+    // {
+    //     convolution_dilation1 = ncnn::create_layer(ncnn::LayerType::Convolution);
+
+    //     // set param
+    //     ncnn::ParamDict pd;
+    //     pd.set(0, num_output); // num_output
+    //     pd.set(1, kernel_w);
+    //     pd.set(11, kernel_h);
+    //     pd.set(2, 1);
+    //     pd.set(12, 1);
+    //     pd.set(3, 1);  // stride_w
+    //     pd.set(13, 1); // stride_h
+    //     pd.set(4, 0);  // pad_w
+    //     pd.set(14, 0); // pad_h
+    //     pd.set(5, bias_term);
+    //     pd.set(6, weight_data_size);
+
+    //     convolution_dilation1->load_param(pd);
+
+    //     // set weights
+    //     if (bias_term)
+    //     {
+    //         ncnn::Mat weights[2];
+    //         weights[0] = weight_data;
+    //         weights[1] = bias_data;
+
+    //         convolution_dilation1->load_model(ModelBinFromMatArray(weights));
+    //     }
+    //     else
+    //     {
+    //         ncnn::Mat weights[1];
+    //         weights[0] = weight_data;
+
+    //         convolution_dilation1->load_model(ModelBinFromMatArray(weights));
+    //     }
+
+    //     convolution_dilation1->create_pipeline(opt);
+
+    //     return 0;
+    // }
```

Comment:
prefer ```#if 0``` .... ```#endif``` macro for commenting large code blocks

Reply:
done!